src/renderRepoCard.js

```diff
@@ -42,6 +47,31 @@ const renderRepoCard = (repo, options = {}) => {
     `
     : "";
 
+  const svgLanguage = `
+    <g transform="translate(30, 100)">
+      <circle data-testid="lang-color" cx="0" cy="-5" r="6" fill="${langColor}" />
+      <text data-testid="lang" class="gray" x="15">${langName}</text>
+    </g>
+  `;
+
+  const svgStars =
+    stargazers.totalCount > 0 &&
```

Comment:
I'm using short-circuit operator instead of ternary because FlexLayout function filters out empty strings